### PR TITLE
[DO NOT MERGE]Update no visa outcome for countries in EEA and Ireland

### PR DESCRIPTION
**Pull request by edwardkerry**
[Trello](https://trello.com/c/VIhHyhO6/913-5-blocked-on-no-deal-decisiondeploy-if-no-deal-check-if-you-need-a-visa)

### CODE REVIEW

File: lib/smart_answer_flows/check-uk-visa/outcomes/outcome_no_visa_needed.govspeak.erb [hunk 1]
```diff
@@ -1,10 +1,25 @@
...
+  <% end %>
 <% end %>
 
 <% content_for :body do %>
 
-  ^When the UK leaves the EU, the rules for how long you can stay without a visa may change. [Check if you need to act now](/guidance/visiting-the-uk-after-brexit) so you can travel as planned.
+  <% if calculator.passport_country_is_ireland? %>
+    Your rights to live, work and study in the UK have not changed now that the UK has left the EU.
+  <% elsif calculator.passport_country_in_eea? %>
+    Your rights to live, work and study in the UK will not change until then.
```
Comment (MuriloDalRi) [inline lib/smart_answer_flows/check-uk-visa/outcomes/outcome_no_visa_needed.govspeak.erb]: I might be reading this sentence out of context but is something missing here?

Comment (edwardkerry) [reply lib/smart_answer_flows/check-uk-visa/outcomes/outcome_no_visa_needed.govspeak.erb]: It comes under the heading
` You do not need a visa to come to the UK before 31 December 2020`

Comment (MuriloDalRi) [reply lib/smart_answer_flows/check-uk-visa/outcomes/outcome_no_visa_needed.govspeak.erb]: Ah okay then

**Comment by benthorner** [comment]
Hi @edwardkerry. Do we still need this PR, or can we close it?

**Comment by edwardkerry** [comment]
There is a chance that this PR will still be used in the future, so I am re-opening it. As soon as I receive final confirmation I will merge or close as appropriate 👍

**Comment by theseanything** [comment]
@edwardkerry Any update on this PR? Can we close it for now if it's been merged soon?

**Comment by ChrisBAshton** [comment]
@theseanything yes we believe this is now superseded by #4952.

**Pull request closed without merging**